model_compression_toolkit/ptq/keras/quantization_facade.py

```diff
@@ -140,15 +142,20 @@ def keras_post_training_quantization(in_model: Model,
                                                target_resource_utilization=target_resource_utilization,
                                                tb_w=tb_w)
 
-        tg = ptq_runner(tg, representative_data_gen, core_config, fw_info, fw_impl, tb_w)
+        float_graph = copy.deepcopy(tg)
+
+        graph_with_stats_correction = ptq_runner(tg, representative_data_gen, core_config, fw_info, fw_impl, tb_w)
+        quantized_graph = quantize_graph_weights(graph_with_stats_correction)
```

Comment:
Can this run only if we use similarity analyzer? (inside the if scope beneath that)